scripts/next_step_assignment_sample_receipt.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python
+from EPPs.common import StepEPP
+from pyclarity_lims.entities import Protocol
+
+
+class AssignNextStepSampleReceipt(StepEPP):
+    """
+    This script checks to see if any of the relevant step UDFs are answered indicating that a manager review is required
+    """
+
+    def _run(self):
+
+        # obtain the actions of the step then creates a StepActions entity for the current step
+        actions = self.process.step.actions
+
+        # obtain the next actions in the step then creates a list of dict for next_actions for the step
+        next_actions = actions.next_actions
+
+        # check to see if review argument flag is present, if 'true' then set all next actions to "review"
+        if self.process.udf['Dry ice remaining in package?'] =='No' or self.process.udf['Container(s) undamaged and sealed?'] == 'No'\
+            or self.process.udf['Samples frozen?'] == 'No' or self.process.udf['>= 50ul sample present in wells or tubes?']=='No':
```

Comment:
This could probably be written in a more concise way but even if you want to be explicit keep only one `or` per line to make it more readable.